cmd/opentelemetry-collector/cmd/ingester/main.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/open-telemetry/opentelemetry-collector/config"
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+	"github.com/open-telemetry/opentelemetry-collector/service"
+	"github.com/spf13/viper"
+
+	jflags "github.com/jaegertracing/jaeger/cmd/flags"
+	ingesterApp "github.com/jaegertracing/jaeger/cmd/ingester/app"
+	"github.com/jaegertracing/jaeger/cmd/opentelemetry-collector/app"
+	"github.com/jaegertracing/jaeger/cmd/opentelemetry-collector/app/defaults"
+	jconfig "github.com/jaegertracing/jaeger/pkg/config"
+	"github.com/jaegertracing/jaeger/plugin/storage"
+)
+
+func main() {
+	handleErr := func(err error) {
+		if err != nil {
+			log.Fatalf("Failed to run the service: %v", err)
+		}
+	}
+
+	info := service.ApplicationStartInfo{
+		ExeName:  "jaeger-opentelemetry-ingester",
+		LongName: "Jaeger OpenTelemetry Ingester",
+		// TODO
+		//Version:  version.Version,
+		//GitHash:  version.GitHash,
+	}
+
+	v := viper.New()
+
+	storageType := os.Getenv(storage.SpanStorageTypeEnvVar)
+	if storageType == "" {
+		storageType = "cassandra"
```

Comment:
Just a note: We should discuss this as part of the 2.0 tracker - as Uber are not necessarily going to stick with cassandra, so possibly there shouldn't be a default.

Reply:
+1 could you please create an issue for it in area/otel?

Reply:
https://github.com/jaegertracing/jaeger/issues/2228